### PR TITLE
LPS-74529 Apply it

### PR DESCRIPTION
**Pull request by rafaprax**

### CODE REVIEW

File: dynamic-data-mapping-form-renderer/src/main/java/com/liferay/dynamic/data/mapping/form/renderer/DDMFormTemplateContextProcessor.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+package com.liferay.dynamic.data.mapping.form.renderer;
+
+import aQute.bnd.annotation.ProviderType;
+
+import com.liferay.portal.kernel.json.JSONObject;
+
+/**
+ * @author Rafael Praxedes
+ */
+@ProviderType
```
Comment (marcellustavares) [inline dynamic-data-mapping-form-renderer/src/main/java/com/liferay/dynamic/data/mapping/form/renderer/DDMFormTemplateContextProcessor.java]: @rafaprax @leoadb  why are we exporting this service, would it be useful to outside of this module?

Comment (rafaprax) [reply dynamic-data-mapping-form-renderer/src/main/java/com/liferay/dynamic/data/mapping/form/renderer/DDMFormTemplateContextProcessor.java]: @marcellustavares We need this to fix an issue in the autosave feature of a form entry.

**Comment by rafaprax** [comment]
Closing to send the fixes in the proper tickets

**Comment by marcellustavares** [comment]
@rafaprax, please consider to move all the code to internal packages first unless really necessary.

**Pull request closed without merging**